src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Stop-AzsDiskMigrationJob.ps1

```diff
@@ -0,0 +1,102 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+
+Code generated by Microsoft (R) PSSwagger
+Changes may cause incorrect behavior and will be lost if the code is regenerated.
+#>
+
+<#
+.SYNOPSIS
+    After a migration job is created, call this command to cancel the migration job.
+
+.DESCRIPTION
+    Cancel a disk migration job.
+
+.PARAMETER Location
+    Location of the resource.
+
+.PARAMETER MigrationId
+    The migration job guid name.
+
+.EXAMPLE
+
+    PS C:\> $migration =New-AzsDiskMigrationJob -Name "mymigrationJob" -Disks $list -location local -TargetShare "\\SU1FileServer.azurestack.local\SU1_ObjStore"
+    PS C:\> Stop-AzsDiskMigrationJob -Location local -MigrationId $migration.MigrationId
+
+#>
+function Stop-AzsDiskMigrationJob
+{
+    [OutputType([Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob])]
+    [CmdletBinding(DefaultParameterSetName='DiskMigrationJobs_Cancel')]
+    param(    
+        [Parameter(Mandatory = $false, ParameterSetName = 'DiskMigrationJobs_Cancel')]
+        [System.String]
+        $Location,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'DiskMigrationJobs_Cancel')]
+        [Alias('MigrationId')]
```

Comment:
Remove MigrationId and all references to $MIgrationId

Reply:
We'd keep what it is look like because it is autogerated. And I don't think it has issue or has any user experience effect.